Marlin/src/lcd/ultralcd.cpp

```diff
@@ -4758,11 +4758,11 @@ void lcd_update() {
 
       #if ENABLED(DOGLCD)  // Changes due to different driver architecture of the DOGM display
         if (!drawing_screen) {
-#if defined(TARGET_LPC1768)
-digitalWrite(P1_4, !digitalRead(P1_4));  //re-arm (was 77 in the old system)
-#else
-digitalWrite(29, !digitalRead(29));  //2560
-#endif           
+          #ifdef TARGET_LPC1768
+            TOGGLE(P1_4);   // Re-ARM (was 77 in the old system)
+          #else
+            TOGGLE(29);     // If it's not LPC1768 then it must be 2560
+          #endif           
```

Comment:
@Bob-the-Kuhn — Is this only inserted into 2.0.x for debugging purposes? It makes me wonder what effect toggling pin "29" has on various other boards and platforms?